internal/pipeline/executor.go

```diff
@@ -246,12 +246,17 @@ func hydrate(p *pkg, hctx *hydrationContext) (resources []*yaml.RNode, err error
 	// TODO(droot): parameterize the sink-mode (hctx.sinkMode) to determine whether to
 	// write it in-place or not. Currently in-place.
 	pkgWriter := &kio.LocalPackageWriter{PackagePath: p.Path()}
+	// we will gather filters from the pipeline
```

Comment:
Such comments is not making things more clear (it's just rephrasing the function call in words). Either expand or remove it.